Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "regex"
-version = "0.1.48"  #:version
+version = "0.1.49"  #:version
```

Comment:
Could you remove the version bump please? I typically do them in separate commits with tags using a script.